core/build.sh

```diff
@@ -139,24 +144,21 @@ do_action clean
 
 do_action configure
 
-# After we have built the necessary internal dependencies, then we can go
-# ahead and build a fat library for external consumption
-if builder_start_action configure:mac; then
-  mkdir -p "$KEYMAN_ROOT/core/build/mac/$BUILDER_CONFIGURATION"
-  builder_finish_action success configure:mac
-fi
+# After we have built the necessary internal dependencies, then we can go ahead
+# and build a fat library for external consumption for mac, and no-op for win
+builder_run_action configure:mac   mkdir -p "$KEYMAN_ROOT/core/build/mac/$BUILDER_CONFIGURATION"
+builder_run_action configure:win   mkdir -p "$KEYMAN_ROOT/core/build/win/$BUILDER_CONFIGURATION"
```

Comment:
Also shortened to use `builder_run_action` instead of `builder_start_action` + `builder_finish_action`